src/drivers/generic/CMakeLists.txt

```diff
@@ -3,3 +3,7 @@
 if (CONFIG_DUMMY_DMA)
 	add_local_sources(sof dummy-dma.c)
 endif()
```

Comment:
Please split this into at least 3 commits:
1) Adding the buffer copier interface
2) Adding the echo reference interface
3) The rest.

Reply:
Split into series of commits.